docs/07-lifecycle.md

```diff
@@ -2,8 +2,54 @@
 ## Development
 ## Submit to Marketplace
 ## Provision
+### App Bundle
+App bundle contains `manifest.json` file, `static/` folder and one or several lambda function bundles.
```

Comment:
We should probably have an AWS version of Hello World manifest here

Reply:
We already have the hello manifest in the docs right? Should I duplicate?